README.md

```diff
@@ -63,17 +63,19 @@ TEST RESULT
 ```
 Usage:
   treon
-  treon [PATH] [--threads=<number>] [-v]
+  treon [PATH] [--threads=<number>] [-v] [--exclude=<path>]...
```

Comment:
Can remove trailing `...`

Reply:
The trailing ... is not a mistake. this is the syntax for letting the argument appear more than once.

Reply:
ah, thanks.